src/django_ixctl/models/ixctl.py

```diff
@@ -96,6 +96,13 @@ class InternetExchange(PdbRefModel):
     updated = models.DateTimeField(auto_now=True)
     urlkey = models.CharField(max_length=255, default=generate_secret, unique=True)
 
+    slug = models.CharField(
```

Comment:
@egfrank lets add a unique constraint for slug, instance pair

Reply:
Oh right thanks for mentioning that!

Reply:
Addressed in https://github.com/fullctl/ixctl/pull/35/commits/3d5acdf4aefdb13d62012dd086c633b1e7660bfb